apps/src/blockly/addons/cdoFieldVariable.js

```diff
@@ -4,7 +4,7 @@ import msg from '@cdo/locale';
 const RENAME_THIS_ID = 'RENAME_THIS_ID';
 const RENAME_ALL_ID = 'RENAME_ALL_ID';
 
-export default class FieldVariable extends GoogleBlockly.FieldVariable {
+export default class CdoFieldVariable extends GoogleBlockly.FieldVariable {
```

Comment:
These class renames are just for standardization/convenience when debugging -- we were renaming these on import to googleBlocklyWrapper to CdoFieldVariable, but the Chrome debugger shows the actual class name, which makes distinguishing them from google blockly difficult.

Reply:
I like this change/convention!